dune/grid/cpgrid/processEclipseFormat.cpp

```diff
@@ -99,12 +103,52 @@ namespace cpgrid
         std::vector<double> coordData;
         ecl_grid.exportCOORD(coordData);
 
-        std::vector<double> zcornData;
-        ecl_grid.exportZCORN(zcornData);
-
         std::vector<int> actnumData;
         ecl_grid.exportACTNUM(actnumData);
 
+        std::vector<double> zcornData;
+        {
```

Comment:
It would be good to have this section as a separate function returning (possibly sanitized) zcornData, to keep the length of this function more managable.